.github/workflows/size-compare.yml

```diff
@@ -0,0 +1,66 @@
+name: Compare Sizes
+
+on:
+  pull_request_target:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this is scary. Could you add a large WARNING banner (comment) at the top of the file so we don't break this later on by adding a checkout step.

Also I suggest linking to this article for more information: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/